docs/src/components/MigrationChanges/MigrationGuideCallout.tsx

```diff
@@ -0,0 +1,32 @@
+import { Alert, Link } from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+import {
+  CURRENT_MAJOR_VERSIONS,
+  PREV_MAJOR_VERSIONS,
+  Framework,
+} from '../../data/frameworks';
+
+export const MigrationGuideCallout = ({ framework }) => {
+  const {
+    query: { platform = 'react' },
+  } = useRouter();
+
+  if (!framework) {
+    framework = platform as Framework;
+  }
+
+  return (
+    <Alert
+      role="none"
+      variation="info"
+      heading={`ui-${framework} v${PREV_MAJOR_VERSIONS[framework]}`}
+    >
+      The ui-{framework} package is currently on version{' '}
+      {CURRENT_MAJOR_VERSIONS[framework]}. Working with version ui-{framework}@
+      {PREV_MAJOR_VERSIONS[framework]}.x or earlier?{' '}
+      <Link href="../../getting-started/migration">
```

Comment:
I noticed this link always redirects to the react site, kinda stumped on how to fix that